pkg/instancetype/instancetype.go

```diff
@@ -988,7 +988,7 @@ func applyMemory(field *k8sfield.Path, instancetypeSpec *instancetypev1beta1.Vir
 		if vmiSpec.Domain.Resources.Requests == nil {
 			vmiSpec.Domain.Resources.Requests = k8sv1.ResourceList{}
 		}
-		podRequestedMemory := instancetypeMemoryGuest.Value() * (1 - int64(instancetypeMemoryOvercommit)/int64(100))
+		podRequestedMemory := instancetypeMemoryGuest.Value() * (1 - int64(instancetypeMemoryOvercommit)/int64(100)) //nolint:gomnd
```

Comment:
Defining constants and expressing through their names what this number means or why it was chosen is a better solution.

Reply:
I agree in general but with this specific example I don't see any benefit in readability by wrapping this in a constant when it's clear from the current line why the value is being used.

Reply:
At the moment we place opinions over linter rules, we break the standard.
We either follow the linter or drop the rule it tries to enforce and reason why it is not a good fit.

If everyone will act like this, the linter will mean nothing as I and others will not be able to trust it doing the job.

Reply:
> I agree in general but with this specific example I don't see any benefit in readability by wrapping this in a constant when it's clear from the current line why the value is being used.

I agree about readability, but I have a problem with breaking rules due to opinions/interpretations.
It is annoying but sometime it is better to just add another constant (or two) to make a linter happy over declaring he is wrong.

Reply:
Honestly, if this is not a valid lint exception, I wonder what could be considered valid.
I agree with the linter rule! But not in this specific case

Reply:
Okay done, I've kept it close by at least so the reader doesn't have to scroll around to find it.